src/site/content/en/lighthouse-best-practices/no-document-write/index.md

```diff
@@ -1,47 +1,61 @@
 
 For users on slow connections,
 such as 2G, 3G, or slow Wi-Fi,
-external scripts dynamically injected via `document.write()`
+third-party scripts dynamically injected via `document.write()`
 can delay the display of main page content by tens of seconds.
-Lighthouse lists out every call to `document.write()`:
+Chrome therefore blocks the execution of these scripts.
+
+## How the Lighthouse `document.write()` audit fails
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/) flags
+all calls to `document.write()`:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="no-document-write.png" alt="Lighthouse audit showing usage of document.write">
-  <figcaption class="w-figcaption">
-    Page uses <code>document.write()</code>.
-  </figcaption>
+  <img class="w-screenshot" src="no-document-write.png" alt="Lighthouse audit showing usage of document.write">
 </figure>
 
-See [Intervening against `document.write()`](https://developers.google.com/web/updates/2016/08/removing-document-write)
-to learn more.
+Lighthouse reports calls to `document.write()` because it's often used to inject
+third-party render-blocking scripts. Chrome won't execute such scripts
+when all the following conditions are met:
+- The user is on a slow connection.
+- The call to `document.write()` is in a top-level document.
+- The script in the `document.write()` call is render-blocking. (Scripts with the `async` or `defer` attributes will still execute.)
+- The script is not hosted on the same site.
+- The script is not already in the browser HTTP cache.
+- The page request isn't a reload.
 
-## How this audit fails
-
-Lighthouse reports every instance of `document.write()` that it encounters.
-Chrome's intervention against `document.write()` only applies
-to render-blocking, dynamically-injected scripts.
-Other uses of `document.write()` may be acceptable.
+See Google's [Intervening against `document.write()`](https://developers.google.com/web/updates/2016/08/removing-document-write)
+article for more information.
 
 {% include 'content/lighthouse-best-practices/scoring.njk' %}
 
 ## Avoid `document.write()`
 
-Review the usage of `document.write()`.
-If the script meets the criteria outlined in the introduction to
-[Intervening against `document.write()`](https://developers.google.com/web/updates/2016/08/removing-document-write),
-Chrome won't execute the injected script.
-These are the calls to `document.write()` that you want to change.
+Remove all uses of `document.write()` in your code. If it's being used
+to inject third-party scripts, try using
+[asynchronous loading](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/adding-interactivity-with-javascript#parser_blocking_versus_asynchronous_javascript)
+instead.
+
+If third-party code is using `document.write()`,
+there are a couple approaches to try:
+- Add an `async` attribute to the script element.
+- Add the script element using DOM APIs
```

Comment:
not sure if this works either.. but it may.  mostly you gotta contact them and say PLZ STOP

Reply:
OK, I just lifted this guidance from the /web article that Kinlan wrote. I'll remove it!